demo/CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
 # Contributing
 
 Contributions are always welcome. Before contributing please read the
-[code of conduct](./CODE_OF_CONDUCT.md) and [search the issue tracker](issues); your issue may have already been discussed or fixed in `master`. To contribute,
+[code of conduct](./CODE_OF_CONDUCT.MD) and [search the issue tracker](issues); your issue may have already been discussed or fixed in `master`. To contribute,
```

Comment:
It looks like are 2 issues with the link here, one of which you've already fixed:
- [x] Capitalize the ".md" file extension
- [ ] Fix the path to the file (it's using a relative path which, when rendered, includes `/demo/` which the CODE_OF_CONDUCT file is not inside of).